src/main/java/org/kainos/ea/db/BandDao.java

```diff
@@ -0,0 +1,33 @@
+package org.kainos.ea.db;
+
+import org.kainos.ea.cli.Band;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BandDao {
+
+    private DatabaseConnector databaseConnector = new DatabaseConnector();
```

Comment:
I've passed the DB connector through so that each time the service calls the dao it will reuse the same connection. I think all this band stuff is from the merge from US004 so I'm not sure if it should be fixed here or do it in the US004 branch?